tests/integration/acp/add_policy/with_invalid_creator_arg_test.go

```diff
@@ -51,22 +49,19 @@ func TestACP_AddPolicy_InvalidCreatorIdentityWithValidPolicy_Error(t *testing.T)
 		},
 	}
 
-	//TODO-ACP: https://github.com/sourcenetwork/defradb/issues/2357
-	testUtils.AssertPanic(t, func() { testUtils.ExecuteTestCase(t, test) })
```

Comment:
todo: You appear to have changed the test so that it no longer tests what the name and description say it does.

The linked ticket https://github.com/sourcenetwork/defradb/issues/2357 suggests that this is because of a sourcehub bug, which presumably is not fixed by this PR.

Is it now impossible to provide an invalid identity to defra? (this seems unlikely to me).  If it is still possible to provide an invalid identity, please preserve this testing what it was.   Otherwise it might be best to delete it and close the ticket.

Please reply to this message with what you have done when you have done it, so I don't forget or lose track of it :)

Reply:
> The linked ticket https://github.com/sourcenetwork/defradb/issues/2357 suggests that this is because of a sourcehub bug, which presumably is not fixed by this PR.

note: This is actually fixed in the `feat/sdk-pkg` branch of SourceHub, and maybe the latest `dev` branch

Reply:
When we bump to latest sourcehub commit on `dev` it I believe fixes this panic.

Reply:
I've restored the tests but limited them to go client only.

Reply:
Thanks Keenan, and thanks for the code-comments explaining why :)